angular-workspace/projects/example-client-app/src/app/app.module.ts

```diff
@@ -55,8 +53,7 @@ import { NavDrawerComponent } from './nav-drawer/nav-drawer.component';
         NimbleCardButtonModule,
         NimbleDialogModule,
         RouterModule.forRoot([
-            { path: '', redirectTo: '/login', pathMatch: 'full' },
-            { path: 'login', component: LoginComponent },
```

Comment:
For similar reasons to some of my other changes, I had to touch the files (to change the error-visible behavior) and opted for the refactor which in this case was removing it.

I'll advocate for removing it. I think it's in practice an annoying extra click when trying to examine the angular app. Also we try and keep the Blazor and Angular versions of these pages similar and unfortunately Blazor cannot support extra views because we can only do hash-based routing for GitHub Pages and our Chromatic hosting workaround and [Blazor doesn't support hash based routing](https://github.com/dotnet/aspnetcore/issues/6175).

So I'd advocate removing the login page and extra routeas is done in this PR. And in a follow-up PR I can update the examples to show off form validation which would actually be handy to have for all of our controls with error-visible states.

Sound like a good approach / concerns that we should have more buy-in?

Reply:
Asked in the team stand-up channel and didn't hear any additional concerns so going with that direction.